.travis.yml

```diff
@@ -13,6 +13,11 @@ before_install:
   - sudo apt-get -qq update
 script: # use the default script section which is equivalent to the following
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia --check-bounds=yes -e 'using Pkg; Pkg.clone("https://github.com/tawheeler/Vec.jl.git"); Pkg.clone("https://github.com/sisl/Records.jl.git"); Pkg.clone("https://github.com/sisl/AutomotiveDrivingModels.jl.git"); Pkg.clone("https://github.com/sisl/AutoViz.jl.git"); Pkg.clone(pwd()); Pkg.build("NGSIM"); Pkg.test("NGSIM"; coverage=true)'
+  - julia --check-bounds=yes -e 'using Pkg; Pkg.add(PackageSpec(url="https://github.com/tawheeler/Vec.jl.git")); 
```

Comment:
```suggestion
  - julia --check-bounds=yes -e 'using Pkg; Pkg.add(PackageSpec(url="https://github.com/sisl/Vec.jl.git")); 
```

Reply:
Also, should all of these url's include the `.git`? I dont think it matters but I noticed you removed them from the install instructions.

Reply:
It does not matter, I will remove it from travis